README.md

```diff
@@ -2,24 +2,22 @@
 
 # services-flake
 
-NixOS-like services for Nix flakes, as a [process-compose-flake](https://github.com/Platonic-Systems/process-compose-flake) module (based on flake-parts).
+Declarative, composable, and reproducible services for your development environment, as a [process-compose-flake](https://github.com/Platonic-Systems/process-compose-flake) module (based on [flake-parts](https://flake.parts)). Supports MacOS and Linux.
```

Comment:
I think this description will be more appropriate. @srid any thoughts?

Reply:
Yes, it is better.

You could still make comparison to NixOS-like services somewhere else.

Reply:
You may want to take this to `doc/index.md` as well.

Reply:
> You could still make comparison to NixOS-like services somewhere else.

That’s still present in the description of the repo

Reply:
Also mentioned “NixOS-like services” here: https://github.com/juspay/services-flake/pull/112/commits/ce5b6163edd7821bd1cb7670524facbcef980e19